lib/util/git/index.ts

```diff
@@ -96,6 +94,14 @@ function checkForPlatformFailure(err: Error): void {
       'GitLab: Commit message does not follow the pattern',
       "Cannot push because commit message does not follow project's push rules",
     ],
+    [
+      ' is not a member of team',
```

Comment:
```suggestion
      ' is not a member of the team',
```

Maybe this would be better? Or are we doing something like this in the final message output:

` userName is not a member of team teamNameVariable`

Reply:
Nope, the first line is the matching git error and the second line is the error message to show. so no changes on first array member. 🙃 

I'll refactor to objects, so it's more clear how it works. 😉